### PR TITLE
Added modifier key to quick PM

### PR DESCRIPTION
**Pull request by poixen**
Ctrl + click on a user name in chat to PM.

### CODE REVIEW

**Comment by ZeldaZach** [comment]
Failed -sad face-

File: cockatrice/src/chatview.cpp [hunk 1]
```diff
@@ -326,7 +326,12 @@ void ChatView::mousePressEvent(QMouseEvent *event)
                     break;
                                       }
                 case Qt::LeftButton :{
-                    emit addMentionTag("@" + userName);
+                    if (event->modifiers() == Qt::ControlModifier) {
+                        const int delimiterIndex = hoveredContent.indexOf("_");
+                        const QString userName = hoveredContent.mid(delimiterIndex + 1);
```
Comment (Daenyth) [inline cockatrice/src/chatview.cpp]: Are you redeclaring these in this scope? I see them defined on line 320-21, are they not visible at this location?

**Comment by Daenyth** [comment]
@poixen from a quick glance at travis it looks like this doesn't work under qt4

```
[ 95%] Building CXX object cockatrice/CMakeFiles/cockatrice.dir/src/userlist.cpp.o
/home/travis/build/Cockatrice/Cockatrice/cockatrice/src/user_context_menu.h: In member function ‘virtual void ChatView::mousePressEvent(QMouseEvent*)’:
/home/travis/build/Cockatrice/Cockatrice/cockatrice/src/user_context_menu.h:31:10: error: ‘void UserContextMenu::openMessageDialog(const QString&, bool)’ is protected
/home/travis/build/Cockatrice/Cockatrice/cockatrice/src/chatview.cpp:332:74: error: within this context
make[2]: *** [cockatrice/CMakeFiles/cockatrice.dir/src/chatview.cpp.o] Error 1
```

**Comment by tooomm** [comment]
Are things like these written down somewhere in a shortcut-table/information document, so people can look and learn about them?

**Comment by Daenyth** [comment]
Nope! The wiki would be a good place for it though

**Pull request merged**